manifests/sssd.pp

```diff
@@ -1,30 +1,32 @@
 # @summary Configure sssd component of tlog
 #
-# @param users
-#   Array of users who will be session logged
+# @param exclude_users
+#   Array of users to exclude from session logging (root is automatically excluded)
+#   This param only takes effect if both profile_session_log::sssd::users and profile_session_log::sssd::groups is empty
```

Comment:
Perhaps a check for misuse and a compiler warning (or failure?) will prevent user issues down the road.

Reply:
Added a warning, if someone tries to set `exclude_users` while either `users` or `groups` is also set.